packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -12,10 +11,8 @@ const fetchWithNetworkError = require('@uppy/utils/lib/fetchWithNetworkError')
 // at all…)
 let socketIo
 function requireSocketIo () {
-  if (!socketIo) {
-    socketIo = require('socket.io-client')
-  }
-  return socketIo
+  // eslint-disable-next-line global-require
+  return socketIo ??= require('socket.io-client')
```

Comment:
I’m confused at to what’s happening here, sorry. Nullish coalescing or something else?

Reply:
Logical nullish assignment operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment
Basically it does the same thing as the old code, I personally find this syntax easier to read – I think it communicates more clearly the intent – but if that feeling is not shared, let's revert this.

Reply:
Thanks!